src/openfermion/ops/_symbolic_operator_test.py

```diff
@@ -151,6 +151,17 @@ def test_init_str(self):
         self.assertIn(correct, fermion_op.terms)
         self.assertEqual(fermion_op.terms[correct], -1.0)
 
```

Comment:
I would really like to see tests like 
my_fermion_operator - FermionOperator(str(my_fermion_operator)) = 0
my_qubit_operator - QubitOperator(str(qubit_operator)) = 0
where you initialize somewhat random things that look like fermion / qubit operators. Since the SymbolicOperator class is more general than just QubitOperator and FermionOperator, it might be a good idea to also have a test that does this for a dummy operator that is initialized with different "actions". But that is less important.